contracts/package.json

```diff
@@ -36,7 +37,8 @@
     "maci-core": "^1.1.2",
     "maci-crypto": "^1.1.2",
     "maci-domainobjs": "^1.1.2",
-    "module-alias": "^2.2.2"
+    "module-alias": "^2.2.2",
+    "solidity-docgen": "^0.6.0-beta.36"
```

Comment:
Think this beta version is ok to use? I see they haven't released a stable v0.6 yet:
https://github.com/OpenZeppelin/solidity-docgen/releases

Overall it's just doc generation... so not too much risk...

Reply:
seems like the  last stable version was 0.5.7 and came out last year: https://www.npmjs.com/package/solidity-docgen?activeTab=versions
since then many beta versions.. 
I'd say it's ok as it's just docs but happy to go with what the team feels more comfortable with

Reply:
I'm ok since it's just doc generation